README.md

```diff
@@ -169,8 +169,48 @@ bazel build --enable_runfiles build_pip_pkg
 TF_NEED_CUDA=1 bazel-bin/build_pip_pkg artifacts
 pip install artifacts/tensorflow_recommenders_addons_gpu-*.whl
 ```
+##### Apple Silicon Support
+**TFRA** Apple Silicon (M1) version requires:
+
+- macOS 12.0.0+
+- Python 3.9
+- tensorflow-macos 2.5.0
+- bazel 4.1.0+
+
+Before installing **TFRA** from source, you need to install tensorflow-macos from Apple. To install the natively supported version of tensorflow-macos, it's recommended to install the [Conda environment](https://github.com/conda-forge/miniforge). 
+
+After installing conda environment, run the following commands in the terminal.
+
+```sh
+# Create a virtual environment
+conda create -n $Your-Environment-Name Python=3.9
+
+# Activate your environment
+conda activate $Your-Environment-Name
+
+# Install TensorFlow macOS dependencies via miniforge
+conda install -c apple tensorflow-deps==2.5.0
+
+# Install base TensorFlow
+python -m pip install tensorflow-macos==2.5.0
+```
+
```

Comment:
add a subtitle 
###### Installing from source

Reply:
The base TensorFlow is installed from PyPi.